content/en/monitors/monitor_types/metric.md

```diff
@@ -190,6 +190,13 @@ For some metrics that report periodically, it may make sense for triggered alert
 
 In most cases this setting is not useful because you only want an alert to resolve once it is actually fixed. So, in general, it makes sense to leave this as `[Never]` so alerts only resolve when the metric is above or below the set threshold.
 
+#### New group delay
+
+Delay the evaluation start by `N` seconds for new groups.
+The option is available with multi-alert mode.
+
+The time (in seconds) to wait before starting alerting, to allow newly created groups to boot and applications to fully start. This should be a non-negative integer. For example, if you are using containerized architecture, setting an evaluation delay prevents your monitor grouped by containers from triggering due to a high resource usage or a high latency when a new container is created. The delay is applied to every new group - which has not been seen in the last 24 hours - and is set to `60 seconds` by default.
+
```

Comment:
```suggestion
Delay the evaluation start by `N` seconds for new groups. The time (in seconds) to wait before starting alerting, to allow newly created groups to boot and applications to fully start. This should be a non-negative integer.

For example, if you are using containerized architecture, setting an evaluation delay prevents your monitor group-by containers from triggering due to high resource usage or high latency when a new container is created. The delay is applied to every new group (which has not been seen in the last 24 hours) and defaults to `60` seconds.

The option is available with multi-alert mode.
```

Reply:
@cswatt Could we keep the former layout ? With **Delay the evaluation start by `N` seconds for new groups.** on top with a line break right after ? To be consistent with other monitor options which are describing it in the same way (Auto Resolve, Evaluation Delay)
Moreover, right phrasing should likely be "monitor grouped by container" rather than "monitor group-by containers"
Should I restore the branch and reopen a PR ?